api/src/devcomp/domain/models/passage-events/PassageEventWithElement.js

```diff
@@ -0,0 +1,26 @@
+import { assertHasUuidLength, assertNotNullOrUndefined } from '../../../../shared/domain/models/asserts.js';
+import { PassageEventWithElementInstantiationError } from '../../errors.js';
+import { PassageEvent } from './PassageEvent.js';
+
+/**
+ * @abstract PassageElementEvent
+ * See PassageEvent for more info.
+ *
+ * This is the base class for all PassageElementEvent.
+ * It's syntaxic sugar for a `PassageEvent` that contains an element, it exists because lots of events happen on a element that we store in the `data` key..
```

Comment:
```suggestion
 * @abstract PassageEventWithElement
 * See PassageEvent for more info.
 *
 * This is the base class for all PassageEventWithElement.
 * It's syntaxic sugar for a `PassageEvent` that contains an element, it exists because lots of events happen on a element that we store in the `data` key..
```